pkg/controller/registry/resolver/operators.go

```diff
@@ -37,6 +40,101 @@ func (s APISet) PopAPIKey() *opregistry.APIKey {
 	return nil
 }
 
+func GVKStringToProvidedAPISet(gvksStr string) APISet {
+	set := make(APISet)
+	// TODO: Should we make gvk strings lowercase to avoid issues with user set gvks?
```

Comment:
Version and group are lower case, kind is camel cased starting capitalized. It looks like kind can be whatever case though: https://github.com/kubernetes/apiextensions-apiserver/blob/7f0e3c78365f6feb94a923557976d4a305de441b/pkg/apis/apiextensions/validation/validation.go#L404

Reply:
I was thinking that for now we can just say it's user error to put a badly formatted "K.v.g" string in. The rest of the logic will just ignore those entries or they won't match the correct "K.v.g"s. What do you think?

Reply:
agreed